editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -2405,18 +2405,18 @@ void Node3DEditorViewport::_notification(int p_what) {
 			}
 
 			Transform t = sp->get_global_gizmo_transform();
+			VisualInstance3D *vi = Object::cast_to<VisualInstance3D>(sp);
+			AABB naabb = vi ? vi->get_aabb() : _calculate_spatial_bounds(sp);
```

Comment:
What does `naabb` mean? If it's for "new AABB", please use `new_aabb` to make the code self-documenting.